content/departments/engineering/teams/core-services/managed-services/platform.md

```diff
@@ -59,15 +74,24 @@ func main() {
 }
 ```
 
-In your implementation of `runtime.Service`, the primary entrypoint `Initialize` provides a `runtime.Contract` that is pre-configured with MSP defaults and offers helpers to connecting to MSP-provisioned resources. For example, to serve your service, you must use `(runtime.Contract).Port`, and to get a securely authenticated PostgreSQL connection, you can use `(runtime.Contract).PostgreSQL.OpenDatabase(...)`.
+In your implementation of `runtime.Service`, the primary entrypoint `Initialize` provides a `runtime.Contract` that is pre-configured with MSP defaults and offers helpers to integrating with MSP-provisioned resources. For example:
+
+- to serve your service, you must use `(runtime.Contract).Port`
+- to get a securely authenticated PostgreSQL connection, you can use `(runtime.Contract).PostgreSQL.OpenDatabase(...)`
+- Sentry error reporting integration for error-level logs is automatically configured when using the provided logger instance
 
 A full example service is available in [`cmd/msp-example`](https://github.com/sourcegraph/sourcegraph/tree/main/cmd/msp-example) that makes use of most MSP functionality.
 
-## Creating and configuring infrastructure for services
+### Service images
 
-Refer to the [sourcegraph/managed-services README](https://github.com/sourcegraph/managed-services/blob/main/README.md) for all documentation for creating configuring MSP deployments and using `sg msp`.
+MSP expects Docker images built for the `linux/amd64` platform.
+If you are using a recent Macbook, this means that images built locally do not work out of the box - you must build images with [the `--platform=linux/amd64` flag](https://docs.docker.com/build/building/multi-platform/#building-multi-platform-images), or build your images and publish in CI.
```

Comment:
Ah, just realized that we have this here! https://github.com/sourcegraph/managed-services/pull/642

Will follow up to align/merge the two